File: 01.JAVA/pg-226-230/pg_227_19.java
import java.util.Scanner;

public class pg_227_19
{
    public static boolean esPrimo(int n) 
    {
        if (n <= 1) 
        {
            return false;
        }
        for (int i = 2; i <= Math.sqrt(n); i++) 
        {
            if (n % i == 0) 
            {
                return false;
            }
        }
        return true;        
    }

    public static void main(String[] args) 
    {

        Scanner enter = new Scanner(System.in);

        int num;

        System.out.println("");
        System.out.println("*******************************************");
        System.out.println("READ ONE NUMBER AND DETERMINE IF IT'S PRIME");
        System.out.println("*******************************************");
        System.out.println("");

        System.out.print("Write an integer number: ");
        num = enter.nextInt();


        if (esPrimo(num)) 
        {
            System.out.println(num + " is a prime number.");
        } 
        else 
        {
            System.out.println(num + " isn't a prime number.");
        }
    }
}

File: 02.PYTHON/pg-226-230/pg_227_19.py
print("")
print("*******************************************")
print("READ ONE NUMBER AND DETERMINE IF IT'S PRIME")
print("*******************************************")
print("")

num = int(input("Write an intenger number: "))

def es_primo(n):
    if n <= 1:
        return False
    for i in range(2, int(n**0.5) + 1):  # solo verificamos hasta la raíz cuadrada
        if n % i == 0:
            return False
    return True

# Verificar y mostrar resultado
if es_primo(num):
    print(f"{num} is a prime number.")
else:
    print(f"{num} isn't a prime number.")